src/forms/inputs/date-input.js

```diff
@@ -55,26 +54,62 @@ const defaultProps = {
 /* Ignore test coverage: */
 /* istanbul ignore next */
 
+// React datepicker expects a date value
+function parseDate(value) {
+  if (!value || value instanceof Date) return value
+
+  return parseISO(value)
+}
+
 function DateInput (props) {
   const {
     input: { name, value, onBlur, onChange },
     meta, // eslint-disable-line no-unused-vars,
     className, // eslint-disable-line no-unused-vars
     ...rest
   } = omitLabelProps(props)
-  const momentValue = value ? moment(value) : null
-  return (
-    <LabeledField { ...props }>
-      <DatePicker 
-        {...{ 
-          id: name,
-          name,
-          selected: momentValue,
-          onBlur: () => onBlur(value),
-          onChange: (val) => onChange(val ? val.toDate() : ''),
-          ...rest
+  const dateValue = parseDate(value)
+  const inputRef = React.useRef()
+  const shouldRefocus = React.useRef(false)
+
+  const CustomInput = React.forwardRef(function TextInput(props, ref) {
+    return (
+      <input type="text" {...props} ref={(el) => {
+          inputRef.current = el
```

Comment:
This is pretty hacky, but it was the only way that I was able to figure out how to pass a ref to the underlying input component

Reply:
Nevermind 😵‍💫 . Don't code at night folks